src/main/java/com/github/matidominati/medicalclinic/model/dto/CreateDoctorCommand.java

```diff
@@ -0,0 +1,19 @@
+package com.github.matidominati.medicalclinic.model.dto;
+
+import com.github.matidominati.medicalclinic.model.entity.Institution;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@AllArgsConstructor
+public class CreateDoctorCommand extends CreateUserCommand {
```

Comment:
Nie powinnismy tutaj dawac  private List<Institution> institutions; jesli juz to 
private List<InstitutionDto> institutions;